test/HiGlassComponentTest.js

```diff
@@ -2927,71 +2934,71 @@ describe('Simple HiGlassComponent', () => {
       div.setAttribute('id', 'simple-hg-component');
 
       hgc = mount(<HiGlassComponent
-        options={{bounded: false}}
+        options={{ bounded: false }}
         viewConfig={noGPSB}
       />,
-        {attachTo: div});
+      { attachTo: div });
 
       hgc.update();
       waitForTilesLoaded(hgc.instance(), done);
     });
 
-    it ("Makes the search box visible", (done) => {
-      let assemblyPickButton = hgc.find('.assembly-pick-button');
-      //expect(assemblyPickButton.length).toEqual(0);
+    it('Makes the search box visible', (done) => {
+      hgc.find('.assembly-pick-button');
+      // expect(assemblyPickButton.length).toEqual(0);
```

Comment:
Either remove both lines or uncomment the `expect`

Reply:
After removing all the lines which shouldn't have side effects, the whole suite failed. So I've put it back in, with a TODO. I think there's a larger issue about order dependency in tests, and this is just one instance.

Reply:
Does the test fail if the asserting is not commented out? I.e.

```javascript
let assemblyPickButton = hgc.find('.assembly-pick-button');
expect(assemblyPickButton.length).toEqual(0);
```

or even better 

```javascript
expect(hgc.find('.assembly-pick-button').length).toEqual(0);
```

Reply:
It did fail. See comments on https://github.com/higlass/higlass/pull/396/commits/22aa7582fc99f3a55348068e57618b203d557b25

I have no objections to this test passing with meaningful assertions - really! - but I don't know why those assertions, and many others, were commented out, and I think either of you would be better placed to get them working again, if that's a requirement.

I'm going to be doing more on Hubmap the next few days and am not going to be able to work  much on this in the immediate future. When I have the time, I'm happy to take care of these house-keeping things that fall through the cracks, but I think I can only do that if we distinguish initial fast PRs whose emphasis is on
- getting lint to pass on more files
- being very conservative about changes to semantics
- leaving a road map of TODOs or filed issues if the intended meaning is unclear.

Or I can do the first pass, and either of you can finish the work in the same PR, but I just don't think the current process is working.

Reply:
I did not meant to intimidate you and I am sorry if my comments made you feel like the current process is not working. I did not check intermediate commits and only looked at your comment, which only mentions the removal of all lines, so I was curious whether the tests fail in general.